db/index.js

```diff
@@ -1,113 +1,80 @@
-const mysql = require('mysql');
-const config = require('./config.js');
+const Pool = require('pg').Pool
 
-const connection = mysql.createConnection(config);
-
-connection.connect((err) => {
-  if (err) {
-    console.log(err, 'ERROR CONNECTING');
-  } else {
-    console.log('connected');
-  }
+const pool = new Pool({
+  user: 'daniel',
+  host: 'localhost',
+  database: 'bookshelf',
+  password: 'thomas',
+  port: 5432,
 });
 
-const insertUser = (user) => {
-  return new Promise((resolve, reject) => {
-    const sql = 'INSERT INTO users (username, avatar) VALUES (?, ?)';
-    const params = [user.username, user.avatar];
-    connection.query(sql, params, (err, result) => {
-      if (err) { reject(err); }
-      resolve(result);
-    });
-  });
-};
-
-const insertReview = (review) => {
-  return new Promise((resolve, reject) => {
-    const sql = 'insert into reviews (user_id, book_id, date, review, rating) values (?, ?, ?, ?, ?)';
-    const params = [review.user_id, review.book_id, review.date, review.review, review.rating];
-    connection.query(sql, params, (err, result) => {
-      if (err) { reject(err); }
-      resolve(result);
-    });
-  });
-};
-
-const getReviews = (id) => {
-  return new Promise((resolve, reject) => {
-    const sql = 'select * from reviews where book_id = ?';
-    connection.query(sql, id, (err, result) => {
-      if (err) { reject(err); }
-      resolve(result);
-    });
+const getTestUserById = (request, response) => {
+  const { id } = request.params;
+  const time = Date.now();
+  pool.query('SELECT * FROM testuser WHERE id = $1', [id], (error, results) => {
+    if (error) {
+      console.log(error);
+    }
+    console.log(`total query time was ${Date.now() - time}ms`);
+    response.status(200).json(results.rows);
```

Comment:
ending the response with a 200 status code when you get an error seems a little wonky

Reply:
I see what you're saying. As it is now the interpreter would not exit the function after logging the error.  I'll try throwing the error. I suppose I could also add an else statement.
Follow-up: I just read a good post about error throwing explaining, among other things, how `throw new Error(error)` has broader browser support than `throw(error)`. 
https://humanwhocodes.com/blog/2009/03/10/the-art-of-throwing-javascript-errors-part-2/